include/pokemon.h

```diff
@@ -443,17 +443,17 @@ struct SpeciesInfo /*0x8C*/
             u32 isFrontierBanned:1; // This species is not allowed to participate in Battle Frontier facilities.
             u32 padding4:14;
             // Move Data
- /* 0x80 */ const struct LevelUpMove *levelUpLearnset;
- /* 0x84 */ const u16 *teachableLearnset;
-            const u16 *eggMoveLearnset;
- /* 0x88 */ const struct Evolution *evolutions;
- /* 0x84 */ const u16 *formSpeciesIdTable;
- /* 0x84 */ const struct FormChange *formChangeTable;
+ /* 0x7E */ const struct LevelUpMove *levelUpLearnset;
```

Comment:
Pointers need to be 4-bit aligned, ending with 0, 4, 8 or C.

Reply:
fixed hopefully